fairytale.md

```diff
@@ -30,10 +32,6 @@ Andrey did not stop him. He remembered that his grandmother had said that there
 
 So, he also boiled dumplings, and together with his son Peter,  tasted the dumplings with this bottle of rice vinegar that had been stored for half a century.
 
-The artifact then opened and inside laid a sword:
```

Comment:
Oh no! I am very sad the sword art is not part of the story any longer! Are you sure you do not want to add it?

Reply:
I like that look but the story and the reader comes first as any good artist would say or something like that